src/components/PreviewPane/index.tsx

```diff
@@ -43,16 +41,7 @@ export const PreviewPane: FC<Props> = ({
     }}
   >
     {layers.map((layer) => (
-      <ConditionalVisibility
-        condition={isCurrentlyVisible({
-          duration: LayerKey.trim in layer ? (layer.trim.end || duration) - layer.trim.start : duration,
-          start: LayerKey.timeline in layer ? layer.timeline.start : 0,
-          time,
-        })}
-        key={layer.id}
-      >
-        {layerRenderers[layer.type]({ compositionDimensions: { height, width }, files, layer, time })}
-      </ConditionalVisibility>
+      <>{layerRenderers[layer.type]({ compositionDimensions: { height, width }, files, layer, time })}</>
```

Comment:
i think the fragment needs a key now

```suggestion
      <Fragment key={layer.id}>{layerRenderers[layer.type]({ compositionDimensions: { height, width }, files, layer, time })}</Fragment>
```

Reply:
ahh thanks. that warning was driving me crazy and for some reason i thought it was within the individual layer components